### PR TITLE
Deprecate primaryUserDefinedNetworkBinding FG

### PR DESCRIPTION
**Pull request by oshoval**
**What this PR does / why we need it**:
Have the relevant changes done by default,
i.e adding the bindings.

**Reviewer Checklist**

> Reviewers are supposed to review the PR for every aspect below one by one. To check an item means the PR is either "OK" or "Not Applicable" in terms of that item. All items are supposed to be checked before merging a PR. 

- [ ] PR Message
- [ ] Commit Messages
- [ ] How to test
- [ ] Unit Tests
- [ ] Functional Tests
- [ ] User Documentation
- [ ] Developer Documentation
- [ ] Upgrade Scenario
- [ ] Uninstallation Scenario
- [ ] Backward Compatibility
- [ ] Troubleshooting Friendly

**Jira Ticket**:

```jira-ticket

```

**Release note**:

```release-note
None
```

### CODE REVIEW

File: controllers/operands/kubevirt.go [hunk 1]
```diff
@@ -521,22 +521,18 @@ func getKVConfig(hc *hcov1beta1.HyperConverged) (*kubevirtcorev1.KubeVirtConfigu
...
-		networkBindings[primaryUDNNetworkBindingName] = primaryUserDefinedNetworkBinding()
-		networkBindings[passtUDNNetworkBindingName] = passtUserDefinedNetworkBinding(sidecarImage)
+	if networkBindings == nil {
+		networkBindings = make(map[string]kubevirtcorev1.InterfaceBindingPlugin)
 	}
-	return networkBindings
 
+	sidecarImage, _ := os.LookupEnv(hcoutil.PrimaryUDNImageEnvV)
+	networkBindings[primaryUDNNetworkBindingName] = primaryUserDefinedNetworkBinding()
+	networkBindings[passtUDNNetworkBindingName] = passtUserDefinedNetworkBinding(sidecarImage)
```
Comment (maiqueb) [inline controllers/operands/kubevirt.go:533]: do we actually want to do this ? AFAICT we don't want to ship passt in 4.18 

@EdDev / @phoracek

Comment (oshoval) [reply controllers/operands/kubevirt.go:533]: was concluded that we keep passt for not few times fwiw

**Comment by oshoval** [comment]
/cc @nunnatsa @maiqueb 
if you can review please

File: controllers/operands/kubevirt.go (hunk 1 above)
Comment (maiqueb) [reply controllers/operands/kubevirt.go:533]: > imho even if we want to remove it, we should not be done on this PR please
best to keep the PR focused on one thing

OK.

**Review by maiqueb — APPROVED** [review]
Thank you.

File: docs/cluster-configuration.md [hunk 2]
```diff
@@ -215,30 +215,6 @@ See [below](#configure-application-aware-quota-aaq) for Application Aware Quota
...
-metadata:
-  name: kubevirt-hyperconverged
-spec:
-  infra: {}
-  workloads: {}
-  featureGates:
-    enableCommonBootImageImport: true
-    deployKubeSecondaryDNS: true
-    enableApplicationAwareQuota: true
-```
```
Comment (nunnatsa) [inline docs/cluster-configuration.md:240]: This section is the example for the whole feature gates section. It shouldn't be removed. Please restore it.

**Review by nunnatsa — COMMENTED** [review]
@oshoval - thanks for the PR! it looks good. Added one comment inline.

File: docs/cluster-configuration.md (hunk 2 above)
Comment (oshoval) [reply docs/cluster-configuration.md]: done

**Review by nunnatsa — APPROVED** [review]
/approve
/retest

**Comment by oshoval** [comment]
Thank you

**Pull request merged**